lib/dependencies/ImportMetaDependency.js

```diff
@@ -0,0 +1,119 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Ivan Kopeykin @vankop
+*/
+
+"use strict";
+
+const NullDependency = require("./NullDependency");
+
+/** @typedef {import("webpack-sources").ReplaceSource} ReplaceSource */
+/** @typedef {import("../Dependency")} Dependency */
+/** @typedef {import("../DependencyTemplate").DependencyTemplateContext} DependencyTemplateContext */
+/** @typedef {import("../Module")} Module */
+/** @typedef {import("../NormalModule")} NormalModule */
+
+/** @enum {number} */
+const ImportMetaProperty = {
+	Url: 0
+};
+
+class ImportMetaDependency extends NullDependency {
```

Comment:
I don't think we need this class anymore when making the other mentioned changes